### DISCUSSION TITLE
Now targeting node 18

### DISCUSSION THREAD

**Discussion by iklein99** [iklein99/homebridge-smartthings#190] [Announcements]
All, the latest version 1.5.12, targets node 18 or better.  Please consider upgrading node.

**Comment by coryabooth** [comment]
If you may wonder "how".  Click the HomeBridge hamburger icon in top right and/or click the triple dot icon to reach the HomeBridge extended settings.

Select "Terminal"

The instructions are actually output in the next screen.
